dp/init.c

```diff
@@ -197,14 +205,120 @@ static inline int port_init(uint8_t port, struct rte_mempool *mbuf_pool)
 	return 0;
 }
 
+#ifdef USE_AF_PACKET
+void
+af_config_network_monitor_interface(uint16_t port_id);
+
+void
+init_af_socks()
+{
+	uint16_t port;
+	struct rte_eth_conf port_conf = port_conf_default;
+	const uint16_t rx_rings = 1, tx_rings = 1;  //tx_rings = rte_lcore_count() - 1;
```

Comment:
Is this needed for later?
`//tx_rings = rte_lcore_count() - 1;`

Reply:
Was it meant to be number of worker cores, to enqueue without a lock?

Reply:
Cleaned up. Multi-queue interface may not be required even in future multi-worker mode.